api/envoy/service/load_stats/v3/lrs.proto

```diff
@@ -17,7 +17,12 @@ option java_multiple_files = true;
 option java_generic_services = true;
 option (udpa.annotations.file_status).package_version_status = ACTIVE;
 
-// [#protodoc-title: Load reporting service]
+// [#protodoc-title: Load reporting service (LRS)]
+
+// Load Reporting Service is an Envoy API to emit load reports. Envoy will initiate a bi-directional stream with a management server.
+// Upon connecting, the management server can send a LoadStatsResponse to a node it is interested in getting the load reports for.
```

Comment:
Maybe link with `:ref:` to the corresponding messages using the `envoy_v3_api_msg` anchors.

Reply:
Please also line wrap around 100 col if possible, thanks.